### DISCUSSION TITLE
Dumb Question

### DISCUSSION THREAD

**Discussion by Luckypaul62** [builtbybel/FlyOOBE#17] [General]
A dumb question for which I appologise. Downloaded extracted and ready to run on my windows 10 desktop.....which file do I click on to run the App? No .exe or .setup....wondering if McAfee antivirus has quarantined  a file or if there is another way to start the App? 

Thank you

**Comment by Belim** [comment]
Are you sure you downloaded the correct executable files from here https://github.com/builtbybel/Flyby11/releases/download/0.15/flybyscript.zip and not the binaries? Also, take a look here https://github.com/builtbybel/Flyby11/issues/10

**Reply by Luckypaul62** [reply]
Yes Belim, that fixed it. It worked like a charm. Thank you for the marvelous software
